param.go

```diff
@@ -45,6 +45,9 @@ type param interface {
 	//
 	// This MAY panic if the param does not produce a single value.
 	Build(containerStore) (reflect.Value, error)
+
+	// GraphNode returns a slice of param(s) represented in graphNodes for the dotgraph
+	GraphNode() []graphNode
```

Comment:
Let's consider making this `DotNodes()` (plural), so that it better describes that a slice is returned.